CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Resolved off-by-one issue with erasing characters in the last column
 - Excessive polling every 100ms with `live_config_reload` enabled
+- Removal of characters before the URL scheme now take UTF8 byte sizes into account
```

Comment:
This is a very technical description of what has been fixed (which is great for devs), but doesn't really help the user understand what has been fixed.

I'd propose something more like this:
`- Unicode characters at the beginning of URLs are now ignored`

Of course this is just a suggestion, feel free to improve on it if you have any ideas.

Reply:
@chrisduerr, just updated the changelog entry